model/host/host.go

```diff
@@ -92,6 +92,11 @@ type Host struct {
 
 	// accrues the value of idle time.
 	TotalIdleTime time.Duration `bson:"total_idle_time,omitempty" json:"total_idle_time,omitempty" yaml:"total_idle_time,omitempty"`
+
+	// managed containers require different information based on host type
+	ContainerID string `bson:"container_id,omitempty" json:"container_id,omitempty"`
+	IsParent    bool   `bson:"is_parent,omitempty" json:"is_parent,omitempty"`
+	ParentID    string `bson:"parent_id,omitempty" json:"parent_id,omitempty"`
```

Comment:
While obvious to someone who is familiar to with the docker functionality, it's not obvious to others. ParentID could also mean something in the context of another project we have. So can you add a documentation note above this line that indicates that this variable stores the ID of the host this container is on.